src/server/env.ts

```diff
@@ -0,0 +1,26 @@
+import { config } from 'dotenv'
+import makeParser from '../lib/makeParser'
+
+config()
+
+interface Env {
+  LOG_LEVEL: string
+  LOG_PRETTY: boolean
+  NODE_ENV: 'development' | 'production' | 'test'
+  PORT: number
+}
+
+const parse = makeParser<Env>({
+  type: 'object',
+  properties: {
+    LOG_LEVEL: { type: 'string', default: 'info' },
+    LOG_PRETTY: { type: 'boolean', default: false },
+    NODE_ENV: { type: 'string', default: 'development' },
+    PORT: { type: 'number', default: 3000 },
+  },
+  required: [],
+})
+
+const { LOG_LEVEL, LOG_PRETTY, NODE_ENV, PORT } = parse(process.env)
```

Comment:
@dakotalillie What do you think of this? it's a bit verbose... but gives me the convenience I was looking for lol. You're either verbose here, or later.

Reply:
I think it's a good idea to destructure all the properties in here! Updated to make it a bit more concise, but I like the move in general.